woof-code/rootfs-skeleton/usr/bin/pupx

```diff
@@ -133,9 +137,15 @@ eval "$RETPARAMS"
 if [ -f ~/.Xwaylandrc ]; then
     if [ "$SCREENSAVERFLAG" = "true" -a "$SCREENSAVERDELAY" -gt 0 ]; then
         sed -i "s%^XWAYLAND_SCREENSAVER_DELAY=.*%XWAYLAND_SCREENSAVER_DELAY=$SCREENSAVERDELAY%" ~/.Xwaylandrc
+		if pidof labwc >/dev/null 2>&1 ; then # could add sway here
```

Comment:
In a Puppy with labwc but without Cage, ~/.Xwaylandrc doesn't exist

Reply:
Cross that bridge when we come to it. I _was_ running pure wayland but hardly anything worked so call it a transitional phase.

Reply:
However, if it's done now (as in #2689 ) at least we are somewhat future proof.

Reply:
Done in fd27bb4